server/server.go

```diff
@@ -3006,6 +3007,8 @@ func (s *Server) startMonitoring(secure bool) error {
 	mux.HandleFunc(s.basePath(HealthzPath), s.HandleHealthz)
 	// IPQueuesz
 	mux.HandleFunc(s.basePath(IPQueuesPath), s.HandleIPQueuesz)
+	// Raftz
+	mux.HandleFunc(s.basePath(RaftzPath), s.HandleRaftz)
```

Comment:
No reason it can't be, just haven't done it yet. Don't know if it's better to keep it HTTP only until we're definitely happy with the shape of the request parameters & response.